BUILD.bazel

```diff
@@ -1248,14 +1240,12 @@ genrule(
         "//:gcs_server",
         "@plasma//:plasma_store_server",
         "//streaming:copy_streaming_py_proto",
-    ] + select({
-        "@bazel_tools//src/conditions:windows": [],  # ignore build streaming for windows
-        "//conditions:default": [":cp_streaming_lib"],
-    }),
+    ],
```

Comment:
I'm not sure exporting symbols works on windows. Is there a similar liner script way like `ray_exported_symbols.lds` or `ray_version_script.lds` on windows? I'm not familiar with windows?

Reply:
Or does this method work on windows-clang?

Reply:
This builds successfully, so I'm assuming it's working. I'm only just trying to get the code to build on Windows, though. The runtime behavior is something someone should look at later.